consensus/state_processing/src/per_epoch_processing/justification_and_finalization_state.rs

```diff
@@ -0,0 +1,114 @@
+use types::{BeaconState, BeaconStateError, BitVector, Checkpoint, Epoch, EthSpec, Hash256};
+
```

Comment:
```suggestion

/// This is a subset of the `BeaconState` which is used to compute justification and finality
/// without modifying the `BeaconState`.
///
/// A `JustificationAndFinalizationState` can be created from a `BeaconState` to compute
/// justification/finality changes and then applied to a `BeaconState` to enshrine those changes.
```

I forgot to add this earlier.